src/devices/yokis.ts

```diff
@@ -0,0 +1,2885 @@
+    // }),
+    numeric({
+        name: 'onTimer',
+        cluster: 'manuSpecificYokisLightControl',
+        attribute: 'onTimer',
+        description: 'Define the ON embedded timer duration in seconds. A `0` value will deactivate the timer',
+        valueMin: 0,
+        valueMax: 3600,
+        valueStep: 1,
+        unit: 's',
+        entityCategory: 'config',
+    }),
+
+    // preOnDelay
+    binary({
+        name: 'ePreOnDelay',
+        cluster: 'manuSpecificYokisLightControl',
+        attribute: 'ePreOnDelay',
+        description: 'Enable (`0x01`) / Disable (`0x00`) PreOn delay',
+        valueOn: ['ON', 0x01],
+        valueOff: ['OFF', 0x00],
+        entityCategory: 'config',
+    }),
+    numeric({
+        name: 'PreOnDelay',
+        cluster: 'manuSpecificYokisLightControl',
+        attribute: 'preOnDelay',
+        description: 'Define the PreOn embedded delay in seconds',
+        valueMin: 0,
+        valueMax: 3600,
+        valueStep: 1,
+        unit: 's',
+        entityCategory: 'config',
+    }),
+
+    // preOffDelay
+    binary({
+        name: 'ePreOffDelay',
+        cluster: 'manuSpecificYokisLightControl',
+        attribute: 'ePreOffDelay',
+        description: 'Enable (`0x01`) / Disable (`0x00`) PreOff delay',
+        valueOn: ['ON', 0x01],
+        valueOff: ['OFF', 0x00],
+        entityCategory: 'config',
+    }),
+    numeric({
+        name: 'PreOffDelay',
+        cluster: 'manuSpecificYokisLightControl',
+        attribute: 'preOffDelay',
+        description: 'Define the PreOff embedded delay in seconds',
+        valueMin: 0,
+        valueMax: 3600,
+        valueStep: 1,
+        unit: 's',
+        entityCategory: 'config',
+    }),
+
+    // Pulseduration
+    numeric({
+        name: 'PulseDuration',
+        cluster: 'manuSpecificYokisLightControl',
+        attribute: 'pulseDuration',
+        description: 'Set the value of ON pulse length',
+        valueMin: 0x0014,
+        valueMax: 0xfffe,
+        valueStep: 1,
+        unit: 'ms',
+        entityCategory: 'config',
+    }),
+
+    // TimeType
+    enumLookup({
+        name: 'TimeType',
+        lookup: timeTypeEnum,
+        cluster: 'manuSpecificYokisLightControl',
+        attribute: 'timeType',
+        description: `Indicates the current Type of time selected that will be used during push button configuration:
+        - 0x00 -> Seconds
+        - 0x01 -> Minutes`,
+        entityCategory: 'config',
+    }),
+
+    // LongOnDuration
+    numeric({
+        name: 'LongOnDuration',
+        cluster: 'manuSpecificYokisLightControl',
+        attribute: 'longOnDuration',
+        description: 'Set the value of the LONG ON embedded timer in seconds',
+        valueMin: 0x00,
+        valueMax: 0x00409980,
+        valueStep: 1,
+        unit: 's',
+        entityCategory: 'config',
+    }),
+
+    // OperatingMode
+    enumLookup({
+        name: 'OperatingMode',
+        lookup: operatingModeEnum,
```

Comment:
Inline the lookups, there is no need to put them on a separate variable.

Reply:
Done - i moved all enumLookup inline (beside one where i am sharing the enums with another element)